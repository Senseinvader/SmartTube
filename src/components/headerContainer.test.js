import React from 'react';
import HeaderContainer from './headerContainer';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';

describe('Header container', () => {
    it('renders search bar child component', () => {
        const wrapper = shallow(<HeaderContainer/>);
        const searchBar = wrapper.find('Connect(SearchBarComponent)');
        const authButton = wrapper.find('Connect(AuthorizationComponent)');
        expect(searchBar.length).toEqual(1);
        expect(authButton.length).toEqual(1);
    });
    it('matches the snapshot', () => {
        const component = shallow(<HeaderContainer/>);
        expect(toJson(component)).toMatchSnapshot();
    })
});