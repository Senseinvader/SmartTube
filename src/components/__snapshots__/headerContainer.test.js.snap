// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header container matches the snapshot 1`] = `
<div
  id="header-container"
>
  <div
    className="search-bar-container"
  >
    <Connect(SearchBarComponent) />
  </div>
  <div
    className="button-container"
  >
    <Connect(AuthorizationComponent) />
  </div>
</div>
`;
